js/forum/src/components/EditUserModal.js

```diff
@@ -15,6 +15,7 @@ export default class EditUserModal extends Modal {
 
     this.username = m.prop(user.username() || '');
     this.email = m.prop(user.email() || '');
+    this.isActivated = m.prop(user.isActivated() || '');
```

Comment:
`this.isActivated = m.prop(user.isActivated() || false);`

`user.isActivated` is a boolean, so let's keep it that way.